### PR TITLE
added phantomjs general js checker

### PR DESCRIPTION
**Pull request by ochriste**
for https://github.com/camptocamp/c2cgeoportal/issues/1848

this checker only check if there is any js error on page load.

### CODE REVIEW

File: c2cgeoportal/views/checker.py [hunk 1]
```diff
@@ -361,3 +362,47 @@ def checker_lang_files(self):
...
+        url = self.request.route_url("mobile")
+
+        driver.implicitly_wait(10)
+        driver.get(url)
+
+        key = u'message'
+        browser_log = driver.get_log('browser')
+        log_message = ''
+        if len(browser_log) > 0:
+            log_message = '<br />nb javascript error(s): %s <br /><br />' % \
```
Comment (sbrunner) [inline c2cgeoportal/views/checker.py]: Do we relay want 2 br between each error lines?

Comment (ochriste) [inline c2cgeoportal/views/checker.py]: it makes things MUCH more readable, trust me, I tried without

Comment (ochriste) [inline c2cgeoportal/views/checker.py]: dont forget, 2 br = one empty line, not 2

Comment (sbrunner) [inline c2cgeoportal/views/checker.py]: Yes, I don't forget it :-), I don't know but it's relay minor :-)

Comment (ochriste) [inline c2cgeoportal/views/checker.py]: well, let say I prefer it like that

File: c2cgeoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl [hunk 2]
```diff
@@ -72,6 +72,22 @@ replace your sencha touch interface by an ngeo interface.
...
+        disabled: [checker_printcapabilities, checker_pdf]
+        check_type:
+            all:
+    ...
+   +         - name: checker_js_generic
+   +           display: PhantomJS generic test
+            main:
+    ...
+   +         - name: checker_js_generic
+   +           display: PhantomJS generic test
```
Comment (sbrunner) [inline c2cgeoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl]: This shouldn't be also in the `CONST_vars.yaml`?

**Comment by sbrunner** [comment]
Just a last question, also that looks good :-)

File: c2cgeoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl (hunk 2 above)
Comment (ochriste) [inline c2cgeoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl]: absolutly!

**Comment by sbrunner** [comment]
Thanks, merge when Travis tests pass :-)

**Comment by sbrunner** [comment]
Looks good for me :-)

**Pull request merged**